sensor/common/scan/scan.go

```diff
@@ -137,19 +157,27 @@ func (s *LocalScan) EnrichLocalImageInNamespace(ctx context.Context, centralClie
 		return nil, errors.Join(ErrNoLocalScanner, ErrEnrichNotStarted)
 	}
 
+	metricLabels := sensorCentralScanMetricsLabel
 	// Throttle the # of active scans.
 	scanLimitSemaphore := s.scanSemaphore
 	// Ad hoc requests have a request ID.
 	if req.ID != "" {
+		metricLabels = sensorAdHocScanMetricsLabel
 		scanLimitSemaphore = s.adHocScanSemaphore
 	}
 
 	semaphoreCtx, cancel := context.WithTimeout(ctx, s.maxSemaphoreWaitTime)
 	defer cancel()
+	images.ScanSemaphoreQueueSize.With(metricLabels).Inc()
```

Comment:
Similar comment `ScanSemaphoreQueueSize` will not be decremented if `Acquire` fails, or am i missing something?